llvm/include/llvm/Target/Target.td

```diff
@@ -1401,6 +1401,14 @@ def FAULTING_OP : StandardPseudoInstruction {
   let isTerminator = true;
   let isBranch = true;
 }
+def FAKE_USE : StandardPseudoInstruction {
```

Comment:
How is this any different than KILL?

Reply:
The only difference in the definition here is the `AsmString`, as with most of the `DBG_` classes which differ only in `AsmString` but are handled differently by MIR passes.

Reply:
I mean if there's a justification for not just re-using kill for this, it needs to be clearly documented here

Reply:
Sure, I've expanded the comment.